clients/spring-boot-starter-camunda-sdk/src/test/java/io/camunda/zeebe/spring/client/config/CredentialsProviderSelfManagedTest.java

```diff
@@ -70,6 +68,7 @@ public class CredentialsProviderSelfManagedTest {
       JWT.create().withExpiresAt(Instant.now().plusSeconds(300)).sign(Algorithm.none());
   @MockBean ZeebeClientExecutorService zeebeClientExecutorService;
   @Autowired ZeebeClientConfigurationImpl configuration;
+  @Autowired private ZeebeClientConfiguration zeebeClientConfiguration;
```

Comment:
@nicpuppa is it required to autowire `ZeebeClientConfiguration`? the implementation is autowired one line above

Reply:
No, I somehow missed it. Thanks 🙏